features/formatter_paths.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Formatter Paths
+
+  Background:
+    Given a file named "features/a.feature" with:
+      """
+      Feature: some feature
+        Scenario: some scenario
+          Given a passing step
+      """
+    And a file named "features/step_definitions/cucumber_steps.js" with:
+      """
+      import {defineSupportCode} from 'cucumber'
+
+      defineSupportCode(({Given}) => {
+        Given(/^a passing step$/, function() {})
+      })
+      """
+
+  Scenario: Relative path
+    When I run cucumber.js with `-f summary:summary.txt`
+    Then the file "summary.txt" has the text:
+      """
+      1 scenario (1 passed)
+      1 step (1 passed)
+      <duration-stat>
+      """
+
+  Scenario: Absolute path
+    When I run cucumber.js with `-f summary:{{{tmpDir}}}/summary.txt`
+    Then the file "{{{tmpDir}}}/summary.txt" is absolute
```

Comment:
I don't like this step. It appears to only be testing that we are using mustache templates correctly. If we remove this step, does the test still fail if we revert the source code change?